bin/ch-run-oci.py.in

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!%PYTHON_SHEBANG%
```

Comment:
What is the PYTHON_SHEBANG macro for?

Reply:
I see it has something to do with slashes.

Reply:
This is so the user can specify the complete shebang line for the Python scripts, without assuming it starts with `/usr/bin/env`.